pages/features.html

```diff
@@ -491,7 +491,7 @@ <h4>Leverage your C# experience to feel right at home</h4>
 			<p>
 				If you're an experienced C# user, Godot offers you first-class support
 				for the .NET platform. Power your game with familiar libraries and give
```

Comment:
```suggestion
				for the .NET platform. Power your games with familiar libraries and give
```
Or change "give it a"